README.md

```diff
@@ -10,9 +10,31 @@ Install via the [AUR](https://aur.archlinux.org/packages/downgrade/).
 
 ## Usage
 
-Downgrade some packages, checking both local cache and the [A.L.A.][ala]:
+Below is the usage script for `downgrade`:
```

Comment:
I would just ditch this sentence now, FWIW.

Reply:
Yes agree, just a re-statement.